### PR TITLE
Testcases using pytest

### LINKED ISSUE #11: write test cases
write test cases using pytest.

### PR DESCRIPTION
**Pull request by shitalmule04**
Fix  #11 
Testcase to find the version.

### CODE REVIEW

**Comment by ganeshhubale** [comment]
@shitalmule04 
- Good work.
- Keep adding new tests.

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@ script:
   - flake8 readit setup.py
   - python setup.py install
```
Comment (ganeshhubale) [inline .travis.yml]: @shitalmule04  
Please remove these lines because now you have done this test.
```
python setup.py install
readit --version
```

**Review by ganeshhubale — CHANGES_REQUESTED** [review]
Please do changes asap.

File: .travis.yml (hunk 1 above)
Comment (shitalmule04) [reply .travis.yml]: @ganeshhubale, done.

**Comment by shitalmule04** [comment]
@ganeshhubale, done.

**Comment by shitalmule04** [comment]
@ganesh, according test-cases point of view `pytest` parameter will search for the files with `test_` labelling. and according to coding python standards point of view it will be not a big deal if we write like `flake8 readit setup.py test_readit.py` in `.travis.yml`

**Review by ganeshhubale — APPROVED** [review]

**Pull request merged**